src/components/expanded-state/UniqueTokenExpandedState.js

```diff
@@ -1,4 +1,5 @@
 import React, { Fragment, useCallback, useMemo } from 'react';
+import { EXPERIMENTAL_AUDIO_PLAYER } from 'react-native-dotenv';
```

Comment:
We use `experimentalConfig` for this. However, I suggest just removing this flag (see some previous comment)